src/tribler-common/tribler_common/logger.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import logging.config
+import sys
+from pathlib import Path
+
+import yaml
+
+LOG_CONFIG_FILENAME = 'logger.yaml'
+
+
+logger = logging.getLogger(__name__)
+
+
+# note: this class is used by src/tribler-common/tribler_common/logger/config.yaml
```

Comment:
👍

Reply:
Fixed the reference to point to the correct file :)